hudi-common/src/test/java/org/apache/hudi/common/functional/TestHoodieLogFormat.java

```diff
@@ -2347,7 +2347,7 @@ public void testDataBlockFormatAppendAndReadWithProjectedSchema(
           new HashMap<HoodieLogBlockType, Integer>() {{
             put(HoodieLogBlockType.AVRO_DATA_BLOCK, 0); // not supported
             put(HoodieLogBlockType.HFILE_DATA_BLOCK, 0); // not supported
-            put(HoodieLogBlockType.PARQUET_DATA_BLOCK, HoodieAvroUtils.gteqAvro1_9() ? 2593 : 2605);
+            put(HoodieLogBlockType.PARQUET_DATA_BLOCK, HoodieAvroUtils.gteqAvro1_9() ? 1802 : 1809);
```

Comment:
Hi @alexeykudinkin, I found that testDataBlockFormatAppendAndReadWithProjectedSchema will fail when we set ${avro.version} = 1.8.2 . The expectedReadBytes expected is 1809 but real read bytes is  1802. Is ${avro.version} must be greater than 1.9? Thank you very much if I can receive your reply.